### ISSUE TITLE
Street Addresses Not Annotated Properly

### ISSUE THREAD

**Issue by greenwoodma** [GateNLP/gateplugin-ANNIE#7]
Unfortunately the gazetteer with street names got removed which means street names are no longer recognized and don't become Address annotations.

**Comment by adam-funk** [comment]
The effect of putting "streets.lst:street:street" back in the lists.def file is to create Location annotations over things like "Main Street" and "Aspen Avenue".  I think that if they are embedded in larger addresses, you get an Address annotation, probably instead of the Location one.  But from looking at the JAPE, I wonder if taking streets.lst out also broke some of the Address functionality.

**Comment by dianamaynard** [comment]
Yes, you get Location with a kind=streetAddress. What I realised was that when you put it back in, you get a lot of spurious hits for things like any mention of the word "road". I have a suspicion that may be why it got taken out temporarily. I fixed that up I think with a bit of JAPEry  (in another project) so I guess we can look at putting it back in with that amendment.
